AllTests.xctestplan

```diff
@@ -9,6 +9,12 @@
     }
   ],
   "defaultOptions" : {
+    "environmentVariableEntries" : [
+      {
+        "key" : "SNOWPLOW_ENDPOINT",
+        "value" : "http:\/\/localhost:9090"
```

Comment:
curious, why not `http://127.0.0.1:9090` as we have in our Environment Variables?

Reply:
also why do we not need to pass in the identifier anymore?

Reply:
localhost is more the defacto standard when accessing the local system.